### DISCUSSION TITLE
Error 403 when creating or modifying shortlinks.

### DISCUSSION THREAD

**Discussion by IOchoam** [YOURLS/YOURLS#3604] [General]
Hi
I am installing yourls YOURLS v1.9.2 on top of IIS.
I can enter the administrative portal and see the short links that come by default. But I can't create new or modify existing shortlinks.

When trying to create a new shortlink, the "shorten the url" button gets stuck, does nothing. The page shows no error. (The same thing happens with the "save" button when I edit a link)

I changed the variable "YOURLS_DEBUG" from false to true but it shows no error so the problem is at the server level.

With the developer mode of the browser I found that the problem is:
server error
403 - Forbidden: Access is denied.
You do not have permission to view this directory or page using the credentials that you supplied.
It is generated in the following Link:
https://subdomain.domain.com/admin/admin-ajax.php?action=add&url=https://google.com/&keyword=gl&nonce=11096cab95

If the URL parameter does not include "https://", the error is not generated and the link is saved in the database, but obviously the shortlink does not work.

I've assigned permissions to the appPool and the "everyone" group on the folder where the yourls files are, but it doesn't seem to fix the problem.

Does anyone have any idea what I can check or do to fix this behavior?
Sincerely

**Comment by dgw** [comment]
You seem to have stopped before one of the most important troubleshooting steps, if you think this is an issue with the server: https://yourls.org/docs/guide/troubleshooting/first-steps#check-server-logs

I'm afraid I won't be of much help, as I've never used IIS and we do not have a current configuration guide for using YOURLS on that platform.

**Comment by IOchoam** [comment]
Hi Dgw.
I already checked the IIS Logs but found the same error code: 403.
I'm going to keep searching for a solution.

Thank you.

**Reply by gashikus** [reply]
Hi. Were you able to find the answer? I can't do it either

**Reply by IOchoam** [reply]
Hi, Gashikus.
No, I wasn't. I installed it in a vm server instead.

If you found a solution, it'll be good to know it.

King regards.
